src/context.cpp

```diff
@@ -83,4 +83,13 @@ namespace oxen::quic
         log::trace(log_cat, "IO context stored connection closed callback");
         conn_closed_cb = std::move(func);
     }
+
+    void IOContext::handle_ioctx_opt(opt::disable_key_verification)
+    {
+        log::warning(
+                log_cat,
+                "IOContext disabling key verification for {}bound connections!",
```

Comment:
Since someone has to explicitly pass this option, it's obviously something they want and it seems a bit weird to warn about it.  Maybe `info` instead?